packages/react/src/List/UnorderedList.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {render, cleanup} from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import {UnorderedList} from './'
+import {axe, toHaveNoViolations} from 'jest-axe'
+
+expect.extend(toHaveNoViolations)
+
+describe('UnorderedList', () => {
+  afterEach(cleanup)
+
+  it('no a11y violations', async () => {
+    const {container} = render(
+      <UnorderedList>
+        <UnorderedList.Item>Automatic security and version updates</UnorderedList.Item>
+        <UnorderedList.Item>GitHub Security Advisories</UnorderedList.Item>
+        <UnorderedList.Item>Code and secret scanning</UnorderedList.Item>
+        <UnorderedList.Item>Dependency review</UnorderedList.Item>
+        <UnorderedList.Item>Automated authentication and identity management</UnorderedList.Item>
+      </UnorderedList>
+    )
+    const results = await axe(container)
+
+    expect(results).toHaveNoViolations()
+  })
+
+  it('renders the correct tag', () => {
+    const mockId = 'mock-id'
+    const {getByTestId} = render(
+      <UnorderedList data-testid={mockId}>
+        <UnorderedList.Item>Automatic security and version updates</UnorderedList.Item>
+      </UnorderedList>
+    )
+    const el = getByTestId(mockId)
+    expect(el.tagName).toBe('UL')
+  })
+
+  it('renders the starting svg', () => {
```

Comment:
Can we have some additional tests for the variant prop too? One for default tag and the other for optional check variant.

Reply:
I was trying to do this but I wasn't sure how to check for specific svg's being loaded. If you have any ideas let me know. I could do a string match but that seems like a cop out

Reply:
Could you use a data-attribute or pass a custom className and pass the name of the icon? It doesn't need to be an equality check in the actual SVG markup or even file path, as that's a project side-effect. Calling the correct name seems important though I feel.

Reply:
I am still not sure how that would work. What are you suggesting the output would be for passing either a custom className or a data-attribute. Those are used for grabbing pieces of the rendered output but I would have to change the component internally to gain access to those in the testing suite and that doesn't seem like the right approach because it is adding unnecessary complication. I do believe there would be a way to see the svg element using this testing suite but I haven't figured out how to see these nested elements yet.

Reply:
It could be a follow up ticket when I have learned more but I can keep digging.